hack/gen-bundle/get-manifests.sh

```diff
@@ -27,17 +39,23 @@ fi
 # operator deployment manifest that doesn't include an init container and
 # volumes for creating install-time resources.
 function downloadOperatorManifests() {
-    curl ${CALICO_BASE_URL}/manifests/ocp/tigera-operator/02-tigera-operator-no-resource-loading.yaml --output ${DEPLOY_DIR}/operator.yaml
-    curl ${CALICO_BASE_URL}/manifests/ocp/tigera-operator/02-role-tigera-operator.yaml --output ${DEPLOY_DIR}/role.yaml
+    curl ${CALICO_BASE_URL}/manifests/ocp/tigera-operator/02-tigera-operator-no-resource-loading.yaml --output ${BUNDLE_DEPLOY_DIR}/operator.yaml
+    curl ${CALICO_BASE_URL}/manifests/ocp/tigera-operator/02-role-tigera-operator.yaml --output ${BUNDLE_DEPLOY_DIR}/role.yaml
+    # The binding is required unlike in earlier bundle generation. The
+    # 'operator-sdk generate bundle' command combines clusterroles bound to service
+    # accounts. The resulting permissions is set to the CSV's
+    # spec.install.clusterPermissions field.
+    curl ${CALICO_BASE_URL}/manifests/ocp/tigera-operator/02-rolebinding-tigera-operator.yaml --output ${BUNDLE_DEPLOY_DIR}/rolebinding-tigera-operator.yaml
 
     # Download the installation CR so that the alm-examples annotation is generated.
-    curl ${CALICO_BASE_URL}/manifests/ocp/01-cr-installation.yaml --output ${DEPLOY_DIR}/01-cr-installation.yaml
+    curl ${CALICO_BASE_URL}/manifests/ocp/01-cr-installation.yaml --output ${BUNDLE_DEPLOY_DIR}/cr-installation.yaml
 }
 
 # Download the installation and tigerastatus CRDs.
 function downloadOperatorCRDs() {
-    curl ${CALICO_BASE_URL}/manifests/ocp/crds/01-crd-installation.yaml --output ${CSV_DIR}/operator.tigera.io_installations_crd.yaml
-    curl ${CALICO_BASE_URL}/manifests/ocp/crds/01-crd-tigerastatus.yaml --output ${CSV_DIR}/operator.tigera.io_tigerastatuses_crd.yaml
+    curl ${CALICO_BASE_URL}/manifests/ocp/crds/01-crd-installation.yaml --output ${BUNDLE_CRD_DIR}/operator.tigera.io_installations_crd.yaml
```

Comment:
Don't we already have these locally in-repo?

Reply:
I think at the time when I added the script the thinking was to get all of the crds from the same spot (our docs site).

Reply:
I think that's fine if that's what we want to do - the RELEASING steps should be updated to make sure this only happens after the docs site has been properly updated (with time for docs.projectcalico.org to regenerate the site and push it to live)